packages/runtime/test-runtime-utils/package.json

```diff
@@ -143,7 +143,8 @@
 	"typeValidation": {
 		"broken": {
 			"ClassDeclaration_MockFluidDataStoreRuntime": {
```

Comment:
I can't believe these mocks are `@alpha`.  We should open a bug to get them internal.

Reply:
AB#8169